tests/codeception/frontend/functional/SignupCest.php

```diff
@@ -10,29 +10,27 @@ class SignupCest
 
     /**
      * This method is called before each cest class test method
-     * @param \Codeception\Event\TestEvent $event
+     * @param \codeception_frontend\FunctionalTester $I
      */
-    public function _before($event)
+    public function _before($I)
```

Comment:
No, `_before` in Cest classes receives an actor object. Starting from 2.1.0 you can specify explicitly what Actror to receive